pytorch_toolkit/ote/ote/modules/config_transformers/nncf_config_transformer.py

```diff
@@ -0,0 +1,115 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from copy import copy
+from mmcv import Config
+
+from ote.utils import load_config
+from ..registry import CONFIG_TRANSFORMERS
+from .base import BaseConfigTransformer
+from .utils import merge_dicts_and_lists_b_into_a
+
+def _convert_value_to_bool(v, key):
+    if isinstance(v, bool):
+        return v
+    if isinstance(v, str):
+        if v.lower().strip() in ("on", "true", "yes", "y"):
+            return True
+        if v.lower().strip() in ("off", "false", "no", "n"):
+            return False
+    elif isinstance(v, int):
+        if v == 1:
+            return True
+        if v == 0:
+            return False
+    err_str = f'Cannot convert to bool the value `{v}`'
+    if key is not None:
+        err_str += ' for the key "{key}"'
+    raise RuntimeError(err_str)
+
+@CONFIG_TRANSFORMERS.register_module()
+class NNCFConfigTransformer(BaseConfigTransformer):
+    POSSIBLE_NNCF_PARTS = ('int8', 'sparsity', 'pruning')
```

Comment:
`int8` should be renamed to `quantization` in this dict and everywhere.

Reply:
I'm not sure that it is a good idea. What about int4 or other types of quantizations -- how they will be named?